build/.mxchip-ci.yml

```diff
@@ -1,28 +1,58 @@
-# Copyright (c) Microsoft. All rights reserved.
-# Licensed under the MIT license. See LICENSE file in the project root for full license information.
 name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
-resources:
-- repo: self
-  clean: true
-phases:
-- phase: iotz
-  queue:
-    name: devicelab-01
+
+jobs:
+- job: Cross_Compile
+  pool:
+    vmImage: ubuntu-16.04
   steps:
   - script: |
-      sudo apt-get update && apt-get install -y \
-      curl \
-      git \
-      python-software-properties \
-      build-essential \
-      pkg-config
-      sudo curl -sL https://deb.nodesource.com/setup_6.x | bash -
-      sudo apt-get install -y nodejs
+      . build_all/mbed5/setup.sh
     displayName: 'setup'
+
   - script: |
-      . build_all/mbed5/build.sh
+      # . build_all/mbed5/build.sh
```

Comment:
why are you commenting out build.sh but also editing the files? Does it not work?

Reply:
I was originally going to put the commands I was using back into build.sh, but figured they didn't add too much to the yml file. Preference?

Reply:
shell script collapsed back into build.sh